Automattic/woocommerce-payments
# Hide helper text related to Storefront theme in Multi-Currency settings when Storefront is not the active theme

Fixes #6182

#### Changes proposed in this Pull Request

This PR will hide the helper text that went with the checkbox to enable the currency switcher on the Storefront theme:

<img width="1074" alt="Screenshot taken on 2023-05-01 at 19 21 55 UTC@2x" src="https://user-images.githubusercontent.com/5512652/235514995-027efea0-7d7d-49d1-8dee-0afc11a95d88.png">

* Updates both the Multi-Currency settings page and onboarding wizard.
* The change just moves the closing parenthesis for the `true` response to be later so that it includes the `div` that holds the helper text.
* Additional _prettier_ formatting was applied.

#### Testing instructions

* Enable the Storefront theme on your site under Themes.
* Navigate to WooCommerce > Settings > Multi-Currency.
* Confirm the option to enable the currency switcher on the Storefront theme shows in the Store Settings.
* Navigate back to Themes and change your theme to one that is not Storefront.
* Navigate back to WooCommerce > Settings > Multi-Currency.
* Confirm the option to enable the currency switcher on the Storefront theme and its helper text no longer show in the Store Settings.

-------------------
